com.sap.cloud.lm.sl.cf.web/src/main/webapp/WEB-INF/security-context.xml

```diff
@@ -25,12 +25,15 @@
             access="#oauth2.hasAnyScope('cloud_controller.write', 'cloud_controller.admin')" method="POST" />
         <sec:intercept-url pattern="/**"
             access="#oauth2.hasAnyScope('cloud_controller.write', 'cloud_controller.admin')" method="DELETE" />
-        <sec:custom-filter ref="resourceServerFilter" before="PRE_AUTH_FILTER" />
-        <sec:custom-filter ref="authFilter" position="LAST" />
+
         <sec:access-denied-handler ref="accessDeniedHandler" />
-        <sec:custom-filter ref="csrfHeadersFilter" after="CSRF_FILTER" />
+
         <sec:custom-filter ref="correlationIdFilter" position="FIRST" />
+        <sec:custom-filter ref="resourceServerFilter" before="PRE_AUTH_FILTER" />
+        <sec:custom-filter ref="authFilter" after="BASIC_AUTH_FILTER" />
+        <sec:custom-filter ref="csrfHeadersFilter" after="CSRF_FILTER" />
         <sec:custom-filter ref="requestSizeFilter" after="PRE_AUTH_FILTER" />
+        <sec:custom-filter ref="rateLimitingFilter" position="LAST"/>
```

Comment:
Will not the "authFilter" fail if there is no user authenticated? 
Meaning this filter will never be reached for non authenticated users and only for authenticated ones, also it will not do anything because since they are authenticated the check for Authorization header will always return true. None of the code will be executed.
Can you check this?
We were supposed to block non-authenticated users.